_config/i18n.yml

```diff
@@ -40,23 +40,23 @@ SilverStripe\Core\Injector\Injector:
       1: null
       2: '`TEMP_FOLDER`'
     properties:
-      ConfigCacheFactory: %$Symfony\Component\Config\ConfigCacheFactoryInterface
+      ConfigCacheFactory: "%$Symfony\\Component\\Config\\ConfigCacheFactoryInterface"
     calls:
       FallbackLocales: [ setFallbackLocales, [['en']]]
-      Loader: [ addLoader, ['ss', %$Symfony\Component\Translation\Loader\LoaderInterface ]]
+      Loader: [ addLoader, ['ss', "%$Symfony\\Component\\Translation\\Loader\\LoaderInterface" ]]
   # Set this translator as our message provider for silverstripe's i18n
   SilverStripe\i18n\Messages\MessageProvider:
     class: SilverStripe\i18n\Messages\Symfony\SymfonyMessageProvider
     properties:
-      Translator: %$Symfony\Component\Translation\TranslatorInterface
+      Translator: "%$Symfony\\Component\\Translation\\TranslatorInterface"
 ---
 Name: textcollector
 ---
 SilverStripe\Core\Injector\Injector:
   SilverStripe\i18n\TextCollection\i18nTextCollector:
     properties:
-      Reader: %$SilverStripe\i18n\Messages\Reader
-      Writer: %$SilverStripe\i18n\Messages\Writer
+      Reader: "%$SilverStripe\\i18n\\Messages\\Reader"
```

Comment:
Can we use single quotes and not have to double escape slash? It's simpler and more standard across all our yml.